lib/zxcvbn_tester.rb

```diff
@@ -0,0 +1,10 @@
+module Zxcvbn
+  class Tester
```

Comment:
I can't tell if we are using this outside of specs, but if we're using it prod, I'd put this in a file called `zxcvbn/tester.rb` in one of the directories in `app/` to match Rails autoloader expectations